2_ESSENTIAL_setting-file/index.md

```diff
@@ -670,7 +670,7 @@ Note that a couple of variables can be [initialized internally](https://ec-jrc.g
 	</textvar>                                                          
 	<textvar name="ThetaForestInit1Value" value="-9999">            
 	<comment>                                                           
-	initial soil moisture content layer 1                                 
+	initial soil moisture content layer 1                                 https://ec-jrc.github.io/lisflood-code/3_step3_preparing-setting-file/
```

Comment:
@StefaniaGrimaldi 
Maybe you added this link by mistake?

Reply:
Yes, that is a mistake.

Reply:
yes, just try to do another commit